fdbserver/BlobManifest.actor.cpp

```diff
@@ -614,10 +615,11 @@ class BlobManifestDumper : public ReferenceCounted<BlobManifestDumper> {
 		loop {
 			state std::vector<BlobManifestFile> allFiles = wait(BlobManifestFile::listAll(writer));
 			int count = BlobManifest::count(allFiles);
+			TraceEvent("BlobManfiestCleanup").detail("Count", count);
```

Comment:
Is this TraceEvent spammy? If not, do we want to capture more details than just count?

Reply:
It happens once every 10 mins. It shouldn't be spammy. so far the count is the most relevant info here for debugging.